nemo/collections/llm/gpt/model/mistral.py

```diff
@@ -35,6 +35,7 @@ class MistralConfig7B(GPTConfig):
     num_query_groups: int = 8
     ffn_hidden_size: int = 14336
     seq_length: int = 32768
+    kv_channels: int = 128
```

Comment:
I think we can remove this line, because `kv_channels` gets set in `__post_init__` [here](https://github.com/NVIDIA/Megatron-LM/blob/c45a41baf2594b203952212187cde6b1335f88ed/megatron/core/transformer/transformer_config.py#L308-L309)
Not setting this preserves the relationship between the variables. I believe queries to `kv_channels` below should trigger the calculation and still output 128)

Reply:
the post init will set it to hidden_size / num_attention_heads, but that's not always correct, for example if the matrix is not square, for example, https://huggingface.co/mistralai/Mistral-Nemo-Base-2407/blob/main/config.json#L8